htdocs/mri/jiv/get_file.php

```diff
@@ -92,6 +94,14 @@
     exit(4);
 }
 
+// If $File contains "DCM_", prefix automatically inserted by the
+// LORIS-MRI pipeline, identify it as $FileExt: "DICOMTAR"
+// Caveat: this is not a real file extension, but a LORIS-MRI
+// convention to identify archived DICOMs
+if (strpos($File, "DCM_") !== false) {
```

Comment:
Probably nitpicking here but shouldn't the validation be performed on the file's base name? And shouldn't we also validate that 'DCM_' occurs at the very beginning of the file's base name?

Reply:
@nicolasbrossard 
it is not at the beginning of the filename. `DCM_` is  usually preceded by the year directory (@driusan had the same comment on another pull request that does similar changes)